src/Console/Output/ConsoleOutputFormatter.php

```diff
@@ -132,12 +145,6 @@ private function printNoFixerStatus(ErrorAndDiffResult $errorAndDiffResult, Conf
                 $this->easyCodingStandardStyle->error($systemError);
             }
         }
-
-        $this->printErrorMessageFromErrorCounts(
-            $errorAndDiffResult->getCodingStandardErrorCount(),
-            $errorAndDiffResult->getFileDiffsCount(),
-            $configuration
-        );
```

Comment:
The lines git has marked as changed are slightly confusing. I didn't actually remove these (they are shown as added further up the file). 

I just extracted the system error code from `printNoFixerStatus()` to a new `printSystemErrors()` method so I could reuse that from `printAfterFixerStatus()`.